### DISCUSSION TITLE
Build rocket chat Android Studio / Windows

### DISCUSSION THREAD

**Discussion by journeym** [RocketChat/Rocket.Chat.ReactNative#4636] [Q&A]
Im sorry, its probably a stupid question, but i cannot understand how should i proceed to build Android app using Android studio.
Currently i downloaded latest release, extracted, and opened android folder in android studio.
I get this error:
[error.txt](https://github.com/RocketChat/Rocket.Chat.ReactNative/files/9882700/error.txt)

What should i do to make this buildable using android studio?

I have read https://github.com/RocketChat/Rocket.Chat.ReactNative/blob/develop/CONTRIBUTING.md but it didnt get me any closer to understanding how to build this.
[yarn.txt](https://github.com/RocketChat/Rocket.Chat.ReactNative/files/9882933/yarn.txt)

I get the below error on yarn android

I also have tried read this https://github.com/RocketChat/Rocket.Chat.ReactNative/discussions/4202, but it looks like just wall of text and not a guide.

**Comment by YICONG0603** [comment]
I'm the same as you, I don't understand what environment it was compiled in, they don't have any documentation, they are just publishing, they tell me that RN0.71 is not supported.
